fixing the UI news showcasing

houssem-yahiaoui opened the pull request:
Fixing the Dyalna news showcasing and proposing a solution for the issue #11

houssem-yahiaoui committed:
fixing the UI news showcasing

youknowriad commented on src/controllers/dyalna.js:
```diff
@@ -5,14 +5,29 @@ const news = (bot, msg) => {
+              'footer': 'Dyalna.com',
+              'footer_icon': 'https://www.dyalna.com/favicon.ico?2'
+            }],
+            'unfurl_links': false
```
I guess this attribute is not useful anymore

youknowriad commented on src/controllers/dyalna.js:
```diff
@@ -5,14 +5,29 @@ const news = (bot, msg) => {
-          convo.say({ text: ` - ${ post.title } https://dyalna.com/post/show/${ post.slug }`, unfurl_links: false })
+          convo.say({
+            'attachments' : [{
+              'fallback': 'Required placeholder for news summary.',
```
Should we use post.title for the fallback or something?

youknowriad replied:
Or maybe we could drop this attribute if not useful

houssem-yahiaoui replied:
as it seems over the docs, its in a way or another a required field but apparently no harm on deleting it ! so yeah let's drop it !

youknowriad commented on .env.sample:
```diff
@@ -1,3 +0,0 @@
-# Generated at https://my.slack.com/services/new/bot
-# API_TOKEN =
-# API_TOKEN_DEV =
```
We should not delete this file :)

youknowriad reviewed (changes requested):
Some small notes ;)

IOAyman reviewed (dismissed):
In addition to the comments by @youknowriad, it would be nice if `npm run lint` had no output 😉

houssem-yahiaoui commented:
Yeah @IOAyman it was the case ! just three warnings coming missing the 3 warnings regarding missing a commas !

houssem-yahiaoui committed:
taking of unneeded attribute.

Taking unneeded attribute and updating code quality regarding eslint comma-dangle warning

houssem-yahiaoui committed:
Create .env.sample

houssem-yahiaoui commented:
@IOAyman @youknowriad i guess it's all good now.
